selenium-system-test/src/test/resources/features/security/Applicant CANNOT proceed without going through security.feature

```diff
@@ -1,11 +1,23 @@
 Feature: Applicant CANNOT proceed without going through security
 
-  Scenario: Applicant CANNOT proceed with application if they have NOT passed 2SV
-    Given an applicant has a confidence level of 200
-    But their credential strength is weak
-    Then they are forced into going through 2SV before being able to proceed with their HtS application
+  Scenario: Applicant CANNOT view the user details page without logging in
+    Given an applicant has NOT logged in
+    When they try to view the user details page
+    Then they are forced to log in
```

Comment:
I'm trying to avoid referring to UI elements in the Gherkin scenarios and, instead, would like to just present the hard business rules. How about "Then they are prompted to log in"?